### PR TITLE
Unescape ampersands for niceness until mako html safe

### PR DESCRIPTION
**Pull request by chrisseto**
ping @lyndsysimon

### CODE REVIEW

**Comment by chrisseto** [comment]
Fixes https://trello.com/c/39t2GSLI

File: website/util/rubeus.py [hunk 1]
```diff
@@ -321,7 +321,7 @@ def _serialize_node(self, node, visited=None, parent_is_folder=False):
             to_expand = False
 
         return {
-            'name': node.title if can_view else u'Private Component',
+            'name': node.title.replace('&amp;', '&') if can_view else u'Private Component',
```
Comment (sloria) [inline website/util/rubeus.py]: Can you add a `TODO` comment to remove this once the HTML-safe changes are made?

**Comment by sloria** [comment]
pass finished. :corn:

**Comment by chrisseto** [comment]
@sloria updated

**Pull request merged**